v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/ExtractionPipeline.scala

```diff
@@ -33,140 +34,149 @@ object ExtractionPipeline {
   )
 
   /**
-    * Convert V2F TSVs to JSON and performing transformations for ingest into the Data Repository.
+    * Convert V2F TSVs to Msg and performing transformations for ingest into the Data Repository.
     */
   def main(rawArgs: Array[String]): Unit = {
     val (pipelineContext, parsedArgs) = ContextAndArgs.typed[Args](rawArgs)
 
-    // extract and convert TSVs to JSON, transform JSON and then save JSON
+    // extract and convert TSVs to Msg, transform Msg and then save Msg
     // FrequencyAnalysis
-    val frequencyAnalysisJsonAndFilePaths = V2FExtractionsAndTransforms.extractAndConvert(
+    val frequencyAnalysisMsgAndFilePaths = V2FExtractionsAndTransforms.extractAndConvert(
       FrequencyAnalysis,
       pipelineContext,
       inputDir = parsedArgs.inputDir,
       relativeFilePath = "*/*.csv"
     )
 
-    val frequencyAnalysisTransformedJsonAndFilePaths = V2FExtractionsAndTransforms
-      .transform(frequencyAnalysisJsonAndFilePaths, FrequencyAnalysis)
+    val frequencyAnalysisTransformedMsgAndFilePaths = V2FExtractionsAndTransforms
```

Comment:
These `val` names are long to the point of being unreadable, can we simplify?

A useful technique can sometimes be to set up local blocks:
```scala
val frequencyAnalysisOutput = {
  val filePaths = ???
  val rawMessages = doThing(filePaths, ???)
  // etc.
}
val otherTableOutput = {
  val filePaths = ???
  val rawMessages = doThing(filePaths, ???)
  // etc.
}
```
Also, if there's a sequence of SCollection transforms that get applied in order for each table, we could pull those into a helper method.

Reply:
Definitely in favor of shortening the variable names. I just tried to create a helper function, but some of the tables need to spit out the extracted and converted (but not yet transformed) data to the variant extraction step. Sooo that seems way messier and at that point I think it might make more sense to just keep the structure as is but rename things and maybe scoot some code around.